repos/fdbt-emailer/src/multi-operator-emailer/email.ts

```diff
@@ -0,0 +1,89 @@
+                            ></path></svg
+                          ><span style="padding-left: 5px;">GOV.UK</span></span
+                        ></a
+                      >
+                    </div>
+                    <div style="width: 66.66%; padding-left: 15px;">
+                      <a
+                        style="color: #ffffff;display: inline-block; margin-bottom: 10px; font-size: 25px; font-weight: 700; text-decoration: none; line-height: 2.5;"
+                        href="/"
+                        id="title_link"
+                        >Create Fares Data service</a
+                      >
+                    </div>
+                  </div>
+                </header>
+                <div>
+                  <p
+                    style="color: #0b0c0c; font-family: Arial,sans-serif; -webkit-font-smoothing: antialiased; font-weight: 400;"
+                  >
+                    One or more multi-operator products require your attention.
+                    Visit the <a href="https://${serviceDomain}/products/multiOperatorProductsExternal#fares-awaiting-your-input">multi-operator dashboard</a> to review your products.
+                  </p>
+                </div>
+              </div>
+            </div>
+          </body>
+        </html>
+`;
+};
+
+export const sendEmails = async (
+    client: SES,
+    serviceDomain: string,
+    serviceEmailAddress: string,
+    emailAddresses: string[],
+): Promise<void> => {
+    const mailTransporter = nodemailer.createTransport({ SES: client });
+
+    const mailOptions: Options[] = emailAddresses.map((emailAddress) => ({
+        from: serviceEmailAddress,
```

Comment:
```suggestion
        from: `no-reply@${serviceDomain}`,
```

With the changes to SES I think this can be updated as above. This might not work for prod until the gov.uk domain is verified in SES